src/posts/introducing-xwrf/index.md

```diff
@@ -0,0 +1,381 @@
+    wrf_projection             object +proj=lcc +x_0=0 +y_0=0 +a=6370000 +b=6...
+Attributes: (12/149)
+    ADAPT_DT_MAX:                    72.0
+    ADAPT_DT_MIN:                    36.0
+    ADAPT_DT_START:                  54.0
+    AERCU_FCT:                       1.0
+    AERCU_OPT:                       0
+    AER_ANGEXP_OPT:                  1
+    ...                              ...
+    WEST-EAST_PATCH_END_STAG:        271
+    WEST-EAST_PATCH_END_UNSTAG:      270
+    WEST-EAST_PATCH_START_STAG:      1
+    WEST-EAST_PATCH_START_UNSTAG:    1
+    W_DAMPING:                       0
+    YSU_TOPDOWN_PBLMIX:              0
+```
+
+Now, say we want to calculate the wind speeds from grid-relative wind vector components using [MetPy](https://unidata.github.io/MetPy/latest/). Because WRF's most commonly used core utilizes a numerically-advantageous [Arakawa-C grid](http://amps-backup.ucar.edu/information/configuration/wrf_grid_structure.html), these wind components are located on grid cell walls and have differing array shapes, which means we need to _destagger_ them onto the grid cell center to perform our calculation (we could also use the already-destaggered earth-relative wind component, but where would be the fun in that? ;) ). Additionally, atmospheric data is most frequently interpreted on isobaric (constant pressure) levels, so we also will use [xgcm](https://xgcm.readthedocs.io/en/latest/) to perform vertical interpolation to our defined pressure levels.
+
+```python
+from metpy.calc import wind_speed
+import xgcm
+
+plevs = np.array([350., 300., 250., 200., 150., 100.]) # in hPa
+wind_speeds_js = []
+
+for ds in [ssp2_ds, ssp5_ds]:
+    for field in ['U', 'V']:
+        ds[field] = ds[field].xwrf.destagger().variable
+    ds = ds.metpy.quantify()
+    _wind_speed = wind_speed(ds.U, ds.V).metpy.dequantify()
+    grid = xgcm.Grid(ds, periodic=False)
+    _wind_speed = grid.transform(
+        _wind_speed,
+        'Z',
+        plevs,
+        target_data=ds.air_pressure.metpy.unit_array.m_as('hPa'),
+        method='log'
+    ).persist()
+    wind_speeds_js.append(_wind_speed)
+```
+
+Now, we can simply subtract the wind speed data of the two simulations from one another, let Dask compute the outcome, and plot the data using `hvplot.quadmesh`.
+
+```python
+ws_difference = (wind_speeds_js[0] - wind_speeds_js[1]).compute()
+max_value = np.max(np.fabs(ws_difference)).item()
+
+plot = ws_difference.hvplot.quadmesh(
+    x='XLONG',
+    y='XLAT',
+    groupby=['Time', 'air_pressure'],
+    widget_location='bottom',
+    title='Wind speed difference between SSPs 245 and 585',
+    cmap='seismic',
+    clim=(-max_value,max_value),
+    clabel='wind speed [m/s]',
+    coastline=True,
+    geo=True,
+    rasterize=True,
+    project=True
+)
+plot
+```
+
+![Animation of wind speed difference over the western U.S. for various times and isobaric levels](/posts/introducing-xwrf/hvplot_output.gif)
```

Comment:
we should put the `gif` in the `public/posts/introductiong-xwrf` directory: https://github.com/xarray-contrib/xarray.dev/tree/main/public/posts

Reply:
Shouldn't it be there already as of https://github.com/xarray-contrib/xarray.dev/pull/343/commits/2b7e893aca76d9f4cd9922d3fc649b402482f951?

Reply:
Oh wait, never mind, messed up git...should be there now as of https://github.com/xarray-contrib/xarray.dev/pull/343/commits/0a5d4ae286c5d0b99a746c13494ab0d59758ffaf